src/lightning/pytorch/callbacks/progress/rich_progress.py

```diff
@@ -491,9 +491,6 @@ def on_validation_batch_end(
         if trainer.sanity_checking:
             self._update(self.val_sanity_progress_bar_id, self.val_batch_idx)
         elif self.val_progress_bar_id is not None:
-            # check to see if we should update the main training progress bar
-            if self.main_progress_bar_id is not None:
-                self._update(self.main_progress_bar_id, self.train_batch_idx + self._val_processed)
             self._update(self.val_progress_bar_id, self.val_batch_idx)
         self.refresh()
 
```

Comment:
Do we need the code in L531 (`_get_train_description`)?

Reply:
Yes I think so. It is about the validation bar below the train bar. I don't think it is related to this PR. I'm not sure though.